### DISCUSSION TITLE
파트 확정에 따른 프로젝트 계획

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#26] [Announcements]
현재 1984프로젝트 플밍 팀원 3명의 파트가 확정이 된 것 같습니다.

@ccmuk58 장현님은 스토리(대화)를 담당
@kyoonge 민경님은 플랫포머(맵, 애니메이션)담당
@fkdl0048 저는 임무(미션, 시스템)담당  

담당 파트가 확정되었지만 서로의 작업은 공유되어야 된다고 생각합니다..!  

그때그때 바쁜 파트나 도움이 필요한 파트가 다르기 때문에 거의 공동작업이라고 생각합니다..! 

우선 해당 파트에 대한 전체적인 기능 리스트를 뽑아보고 테스트로 만들고 같이 이야기하면서 수정해보면 좋을 것 같습니다.

*완벽할 수 없습니다.. 생각 나는 대로 뽑으면 좋을 것 같아요*

ex)

## 임무 기능 리스트?

- [ ] 비슷한 레퍼런스 찾아보기(discussion에 공유 후 기획이랑 상의)
- [ ] 필요 기능 리스트 뽑기
- [ ] ~~기능 공부해보기
- [ ] 프로포 타입 만들기

or

- [ ] 마우스 클릭 이벤트 만들기
- [ ] 미션 베이스 클래스 설계
- [ ] 프로토 타입 미션 설계
- [ ] ...등등

대략적인 틀을 잡고 싶은데 처음부터 너무 크게 잡아야 할지 작게 잡아야 할지 어렵네요.,

같이 이야기 해보면서 정리하면 좋을 것 같습니다..

일단 작업 리스트나 공부해야할 리스트를 issue쪽에 작성해주세요!

**Comment by ccmuk58** [comment]
확인했습니다!
툴 제작?? 같은게 완전 처음이라 앞이 깜깜 하네요,,,,

**Reply by fkdl0048** [reply]
처음에는 생각나는 대로 적고 계속 수정해 나가면 됩니다..!  

처음부터 완벽하게 적을 필요가 절대 없습니다..! 완벽할 수도 없고..

두 가지 항목 땜에 공유를 하고 싶은 것인데..  

1. 작업 내용이 추적 가능하다. (나중에 뭘 작업했는지, 언제 했는지 추적이 가능해지면 매우 유용..)
2. 작업 내용을 팀원들이 알 수 있다 (같은 맥락이지만 작업이 공유 안되는 팀은 퍼포먼스가 떨어지기 때문에..)

**Comment by kyoonge** [comment]
서로 작업내용 공유하는거 좋슴다
저도 써넣어볼게요!
